README.md

```diff
@@ -22,16 +22,6 @@ This plugin interfaces with the Splunk HTTP Event Collector:
       check_index false
       source {TAG}
       sourcetype fluent
-
-      # TIMESTAMP: key1="value1" key2="value2" ...
-      time_format unixtime
-      format kvp
-
-      # Memory buffer with a short flush internal.
```

Comment:
These parameters are actually quite valid, as this plugin is a buffered output plugin so these parameters configure the buffering.
http://docs.fluentd.org/v0.12/articles/buffer-plugin-overview
http://docs.fluentd.org/v0.12/articles/output-plugin-overview